### DISCUSSION TITLE
Semi Pro 有没有预估发布时间呢?

### DISCUSSION THREAD

**Discussion by liect** [DouyinFE/semi-design#893] [Q&A] [answered]
### Which Component Need Enhancement 期望新增功能的组件
- Semi Pro

### Feature Description 功能描述
看到Semi Pro的描述 "Design to Code" ,Semi Pro 的目标似乎是低代码平台?
会有类似AntPro的那种重型业务组件吗?
Semi Pro 预估会什么时候发布呢?

**Comment by shijiatongxue** [comment, marked as answer]
1. Semi Pro 没有重型组件，类似于 AntPro 的业务组件不是我们考虑的范围。未来可能会增加常见的通用组件（对 Semi 基础组件的组合），但和业务关系也不是很大
2. 还在研发中
